### PR TITLE
Add missing params for /calendar endpoint

### PR DESCRIPTION
**Pull request by ChemicalRocketeer**

### CODE REVIEW

File: data/webapi/endpoints/calendar.yaml [hunk 1]
```diff
@@ -3,5 +3,15 @@ endpoints:
...
+      params:
+        query:
+          - name: start
+            type: timestamp
+            desc: |
+              The first date to retrieve data for
+          - name: end
+            type: timestamp
+            desc: |
+              The last date to retrieve data for
```
Comment (umitanuki) [inline data/webapi/endpoints/calendar.yaml]: Good catch. Is it inclusive or exclusive boundary?

Comment (ChemicalRocketeer) [reply data/webapi/endpoints/calendar.yaml]: I don't know

Comment (ChemicalRocketeer) [reply data/webapi/endpoints/calendar.yaml]: updated @umitanuki

Comment (umitanuki) [reply data/webapi/endpoints/calendar.yaml]: thx

**Pull request merged**